exp-models/addon/models/experiment.js

```diff
@@ -139,6 +139,21 @@ const Experiment = DS.Model.extend(JamModel, {
         maxDays = maxDays || Number.MAX_SAFE_INTEGER;
         return minDays <= age && age <= maxDays;
     },
+    isOldEnough(participant) {
```

Comment:
Yup, we're thinking on similar lines.

- We try to avoid changing the meaning of methods that are already used because it can break things... (esp since "in range" would be 0, and we don't want a boolean method returning a falsy value for "yes")
- But a helper function would abstract away the code. -1 for below, 0 for in, 1 for above: this follows the rules for sort functions, because someday it would give an easy way to sort the list of children by eligibility without much extra work.... ;)